src/N3DataFactory.js

```diff
@@ -188,9 +187,8 @@ export class DefaultGraph extends Term {
 // ## DefaultGraph singleton
 DEFAULTGRAPH = new DefaultGraph();
 
-
 // ### Constructs a term from the given internal string ID
-export function termFromId(id, factory) {
+export function termFromId(id, factory, nested) {
```

Comment:
@jeswr In what cases should `nested` be true? Is it the same as `rdfStar` basically?

Reply:
It is really just an *internal* parameter that is used when recursively calling `termFromId` on `quad` terms within other quads.

The only place that parameter should *ever* be used is https://github.com/rdfjs/N3.js/blob/56cc184f78761db27751aa4a8489239e0c287666/src/N3DataFactory.js#L224-L229

Reply:
I've added a comment clarifying this

Reply:
So only by itself; gotcha.